public_html/wp-content/mu-plugins/blocks/source/blocks/organizers/block-content.js

```diff
@@ -79,15 +79,15 @@ export class BlockContent extends Component {
 		return (
 			<PostList
 				{ ...this.props }
-				className="wordcamp-organizers-block"
+				className="wordcamp-organizers__posts"
 			>
 				{ posts.map( ( post ) => /* Note that organizer posts are not 'public', so there are no permalinks. */
 					<div
 						key={ post.slug }
-						className={ classnames( 'wordcamp-organizer', 'wordcamp-organizer-' + post.slug.trim() ) }
+						className={ classnames( 'wordcamp-organizers__post', 'has-slug-' + post.slug.trim() ) }
```

Comment:
`MIGHT` - I wonder if this should be `'has-slug', 'slug-{slug}'`?

Maybe that's premature though, since we don't have a use case for targeting elements that `has-slug` rather than individual ones? But maybe site designers would have a use case in their custom CSS? And if we do eventually need that, we'll have to plan for it now to avoid back-compat complications.

`SHOULD` - I used single quotes in the example above b/c GitHub/markdown won't let me escape backticks, but using JS string interpolation rather than `+` seems more readable.

Reply:
> using JS string interpolation rather than + seems more readable.
👍 Will do that.

>I wonder if this should be 'has-slug', 'slug-{slug}'?
Why not just use `slug-{slug}`, only purpose of `has` adjective here is to improve readability. 
Also,
> Because a modifier class name is not contextualized to a specific component, it should always be written in stylesheets as accompanying the component being modified (.components-panel.is-opened).
Given this line, I would think that you are not supposed use modifier class name on its own. If some would want to target `has-slug` they would hopefully use `wordcamp-organizers__post` instead. What do you think?

Reply:
🤔 

Yeah, I guess the slug is really a property of the post, rather than a state, so it probably shouldn't have the `is/has` prefix. `slug-{slug}` makes sense to me.

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/123/commits/be41c3ddc196715dc6a4b2a237f9a40bc62ec458